sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/OperationalLevelTimeoutExample.java

```diff
@@ -0,0 +1,134 @@
+
+/**
+ * This example shows how to start using the Azure Storage Blob SDK for Java.
+ */
+public class OperationalLevelTimeoutExample {
+
+    /**
+     * Entry point into the basic examples for Storage blobs.
+     *
+     * @param args Unused. Arguments to the program.
+     * @throws IOException If an I/O error occurs
+     * @throws RuntimeException If the downloaded data doesn't match the uploaded data
+     */
+    public static void main(String[] args) throws IOException {
+
+        /*
+         * From the Azure portal, get your Storage account's name and account key.
+         */
+        String accountName = SampleHelper.getAccountName();
+        String accountKey = SampleHelper.getAccountKey();
+
+        /*
+         * Use your Storage account's name and key to create a credential object; this is used to access your account.
+         */
+        StorageSharedKeyCredential credential = new StorageSharedKeyCredential(accountName, accountKey);
+
+        /*
+         * From the Azure portal, get your Storage account blob service URL endpoint.
+         * The URL typically looks like this:
+         */
+        String endpoint = String.format(Locale.ROOT, "https://%s.blob.core.windows.net", accountName);
+
+        /*
+         * Set up a HttpPipelinePolicy that sets duration timeout per call of 5 seconds.
+         */
+        HttpPipelinePolicy mockPolicy = new HttpPipelinePolicy() {
+            @Override
+            public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineNextPolicy next) {
+                return next.process().delayElement(Duration.ofSeconds(5L));
+            }
+
+            public HttpPipelinePosition getPipelinePosition() {
+                return HttpPipelinePosition.PER_CALL;
+            }
+        };
+
+        /*
+         * Create a BlobServiceClient object that wraps the service endpoint, credential, policy with
+         * timeout per call, and a request pipeline.
+         */
+        BlobServiceClient storageClient = new BlobServiceClientBuilder()
+            .endpoint(endpoint)
+            .credential(credential)
+            .addPolicy(mockPolicy)
+            .buildClient();
+
+        /*
+         * This example shows how to pass timeouts in call operations.
+         */
+
+        /*
+         * Create a client that references a to-be-created container in your Azure Storage account. This returns a
+         * ContainerClient object that wraps the container's endpoint, credential and a request pipeline (inherited from storageClient).
+         * Note that container names require lowercase.
+         */
+        BlobContainerClient blobContainerClient = storageClient.getBlobContainerClient("myjavacontainerbasic" + System.currentTimeMillis());
+
+        /*
+         * Create a container in Storage blob account with a timeout duration of 3 seconds, below the timeout duration
+         * passed in the policy. This will trigger a timeout exception.
```

Comment:
nit: This should say a timeout exception will occur if the blob create container call takes longer than 3 seconds to complete as this could not throw if it took 2 seconds